samples/Management/PowerShell/AddingSchematoSchemagroups/Addingschematoschemagroup.ps1

```diff
@@ -0,0 +1,45 @@
+
+#Step 1 - Pass in AD Application details below to be able to fetch authorization token against legitimate Service Principal. 
+$Fields = @{
+    grant_type    = "client_credentials"

```

Comment:
Can you use < and > in placeholder here as well?

Reply:
@serkantkaraca , grant_type has static value to be "client_credentials" in this flow, hence I had kept it without placeholder. Do you think I should use placeholders?

Reply:
My bad. No need for placeholder in that case.